pkg/payload/payload.go

```diff
@@ -103,14 +103,23 @@ type Update struct {
 }
 
 func LoadUpdate(dir, releaseImage string) (*Update, error) {
```

Comment:
this function did 1) load metadata 2) start using it.
not this function does 1) validate dir, 2) load image reference 3) create an payload object 4) load tasks 

I think this not the ideal direction.. feel free to split the image referece and tasks, but don't remove the `loadUpdatePayloadMetadata` ??

Reply:
ok